spec/models/order_detail_spec.rb

```diff
@@ -475,13 +474,14 @@ def create_price_policy(params)
                order_detail: order_detail_without_price_policy,
               )
 
-        Timecop.travel(2.days.from_now) do
-          order_details.each do |order_detail|
-            order_detail.change_status!(OrderStatus.find_by_name("In Process"))
-            order_detail.change_status!(OrderStatus.find_by_name("Complete"))
-            order_detail.reload
-          end
+        current_time = Time.current
```

Comment:
This seems like something we could wrap up in our own helper and keep a block API similar to what Timecop had. Something like:

```ruby
def travel_and_return(interval)
  current_time = Time.current
  travel(interval)
  yield
  travel(current_time)
end
```

We could also have a `travel_to_and_return` if we need it. I'm up for different names.